source/dyaml/loader.d

```diff
@@ -82,23 +82,25 @@ struct Loader
 
         /** Construct a Loader to load YAML from a string.
          *
-         * Params:  data = String to load YAML from. The char[] version $(B will)
-         *                 overwrite its input during parsing as D:YAML reuses memory.
+         * Params:
+         *   data = String to load YAML from. The char[] version $(B will)
+         *          overwrite its input during parsing as D:YAML reuses memory.
+         *   filename = The filename to give to the Loader, defaults to `null`
          *
          * Returns: Loader loading YAML from given string.
          *
          * Throws:
          *
          * YAMLException if data could not be read (e.g. a decoding error)
          */
-        static Loader fromString(char[] data) @safe
+        static Loader fromString(char[] data, string filename = null) @safe
```

Comment:
should we default to `"<unknown>"` instead of `null`? (it's the default in Loader right now, though I'm not a big fan of hardcoding it)

Reply:
Fixed